extensions/wikia/AdEngine/ResourceLoaders/AdEngine2Resource.class.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class AdEngine2Resource {
+	const RESOURCE_DATE_FORMAT = 'Y-m-d';
+
+	static public function register($name, $class) {
```

Comment:
I'm not a fan, but I think we should this:
https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP
as coding convention for all our PHP code